Runtime/Scripts/GltfImport.cs

```diff
@@ -2047,6 +2055,7 @@ async Task
                         accessorData[i] = ads;
                         break;
                     }
+#if UNITY_ANIMATION
```

Comment:
This change would break [mesh instancing](https://github.com/KhronosGroup/glTF/blob/master/extensions/2.0/Vendor/EXT_mesh_gpu_instancing/README.md) support, where translation/scale/rotation buffers are required as well.

Reply:
Mhmm, the problem is that `GetVector3Job` and `GetVector4Job` are in `UNITY_ANIMATION`.
Maybe I can find out a way of allowing these methods to exist outside of `UNITY_ANIMATION` ?

Reply:
Turns out, I was able to just move the two methods out of the `UNITY_ANIMATION`. It's possible they just moved in accidentally because maybe `GetAnimationTimes` was written first and then the `GetVector3Job` and `GetVector4Job` next to it moved into the preprocessor accidentally?